packages/plugin-ngrok/README.md

```diff
@@ -0,0 +1,43 @@
+This package contains a Shopify CLI plugin that enables the creation of ngrok tunnels from `shopify app dev`, 
+allowing previews from any device.
+
+## Requirements
+
+- Shopify CLI
+- [ngrok](https://dashboard.ngrok.com/signup) account
+
+## Installation
+
+Right now, it's installed by default with the Shopify CLI. But it could be manually installed with:
+
+```bash
+shopify plugins install ngrok
+```
+
```

Comment:
Since these steps apply to all the plugins, what about having documentation for all the plugins? Maybe at `docs/plugins.md`?